### PR TITLE
[READY] New Bodysuits, Shrouds and Envirosuits, Universal Left Handed Consistency, and Gun Themed Barsigns

### PR DESCRIPTION
**Pull request by JennaSilverShark**
## About The Pull Request
New flashy skintight suits to wear under your voidsuits, finally all the tools face the same way, and I threw in a couple new barsigns because I was bored.

dreammaker_40g8fUIhop
dreammaker_kNydWIEWlh
dreammaker_5grQsFfsCw
FW0K4xUwYb

## Changelog
:cl:JennaSilver
add: Added EMT, Hazard, Security, Advanced Security, Security Command, Exploration, Exploration Command, Command and Central Command Bodysuits
fix: Left Handed Consistency for all weapons and items
add: A couple bar signs that I made for fun
/:cl:

### CODE REVIEW

**Comment by Zandario** [comment]
These look really great! (thooo what about other species?)

**Comment by JennaSilverShark** [comment]
> These look really great! (thooo what about other species?)

I guess I could give it a try, should work fine for most of the humanoid races though.

**Comment by nik707** [comment]
conflicts

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -712,7 +712,35 @@ datum/gear/uniform/fiendsuit
 datum/gear/uniform/fienddress
 	display_name = "Fiendish Dress"
 	path = /obj/item/clothing/under/fienddress
-	
+
 datum/gear/uniform/leotard
 	display_name = "Black leotard"
 	path = /obj/item/clothing/under/leotard
+
+datum/gear/uniform/bodysuitemt
```
Comment (JennaSilverShark) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:720]: Now that I'm sober I can make these fucking names consistent too

**Comment by JennaSilverShark** [comment]
dreammaker_l7Sdpcatjb
And with that I think we're done here.

**Comment by nik707** [comment]
holy fuck so many emails REAEE

**Comment by JennaSilverShark** [comment]
> holy fuck so many emails REAEE

Sorry, still learning the ropes but this should be ready for real now. This was a fun project, and the next one should be even more fun.

**Comment by JennaSilverShark** [comment]
Merge this shit so I can make more already reee

**Comment by JennaSilverShark** [comment]
dreammaker_apdFBhScv9
dreammaker_L9QmcpAIga
Xenolove ❤️

**Comment by JennaSilverShark** [comment]
Closing to split my branches properly and fix conflicts.

**Comment by Keekenox** [comment]
Why is bar signs part of a pr for suits?

**Pull request closed without merging**